js/layout-javascript/agenda-code.js

```diff
@@ -1733,14 +1729,14 @@ DynamicList.prototype.animateDateForward = function($nextDateElement, nextDateEl
 };
 
 // animates cards forward
-DynamicList.prototype.animateAgendaForward = function(nextAgendaElement, nextAgendaElementWidth) {
+DynamicList.prototype.animateAgendaForward = function(nextAgendaElement, nextAgendaElementWidth, animationSpeed) {
   var _this = this;
 
   return new Promise(function(resolve) {
     _this.$container.find('.agenda-cards-wrapper').animate({
       scrollLeft: '+=' + nextAgendaElementWidth
     },
-    _this.ANIMATION_SPEED,
+    animationSpeed,
```

Comment:
There's no need to change existing callers. Just use `_this.ANIMATION_SPEED` as fallback. This way, the `animationSpeed` parameter is optional.

```suggestion
    typeof animationSpeed === number ? animationSpeed : _this.ANIMATION_SPEED,
```